shell/platform/android/io/flutter/embedding/android/FlutterActivity.java

```diff
@@ -187,14 +182,56 @@ public Intent build(@NonNull Context context) {
 
   @Override
   protected void onCreate(@Nullable Bundle savedInstanceState) {
+    switchLaunchThemeForNormalTheme();
+
     super.onCreate(savedInstanceState);
+
     configureWindowForTransparency();
     setContentView(createFragmentContainer());
-    showCoverView();
     configureStatusBarForFullscreenFlutterExperience();
     ensureFlutterFragmentCreated();
   }
 
+  /**
+   * Switches themes for this {@code Activity} from the theme used to launch this
+   * {@code Activity} to a "normal theme" that is intended for regular {@code Activity}
+   * operation.
+   * <p>
+   * This behavior is offered so that a "launch screen" can be displayed while the
+   * application initially loads. To utilize this behavior in an app, do the following:
```

Comment:
while the flutter application initially loads right? Otherwise people will ask why I don't need to do this with my normal android app.

Reply:
People do need to do this in their normal Android app.